pallets/bridged-asset/src/tests.rs

```diff
@@ -1,224 +1,226 @@
-        assert_ok!(Asset::do_mint(asset_id, &bob, 500.into()));
-        assert_ok!(Asset::transfer(
-            Origin::signed(alice.clone()),
-            asset_id,
-            bob.clone(),
-            250.into()
-        ));
-
-        assert_eq!(
-            Account::<MockRuntime>::get(&asset_id, &alice).free,
-            250.into()
-        );
-        assert_eq!(
-            Account::<MockRuntime>::get(&asset_id, &bob).free,
-            750.into()
-        );
-        assert_eq!(TotalIssuance::get(&asset_id), 1000.into());
-    });
-}
-
-#[test]
-fn transfer_should_raise_insufficient_balance() {
-    new_tester().execute_with(|| {
-        let asset_id = H160::zero();
-        let alice: AccountId = Keyring::Alice.into();
-        let bob: AccountId = Keyring::Bob.into();
-
-        assert_ok!(Asset::do_mint(asset_id, &alice, 500.into()));
-        assert_ok!(Asset::do_mint(asset_id, &bob, 500.into()));
-
-        assert_noop!(
-            Asset::transfer(
-                Origin::signed(alice.clone()),
-                asset_id,
-                bob.clone(),
-                1000.into()
-            ),
-            Error::<MockRuntime>::InsufficientBalance,
-        );
-    });
-}
-
-// In theory this case will never occur since Sum(Account.free) == TotalIssuance and thus
-// overflow should be impossible
-#[test]
-fn transfer_should_raise_overflow() {
-    new_tester().execute_with(|| {
-        let asset_id = H160::zero();
-        let alice: AccountId = Keyring::Alice.into();
-        let bob: AccountId = Keyring::Bob.into();
-
-        let account = AccountData { free: U256::one() };
-        Account::<MockRuntime>::insert(&asset_id, &alice, &account);
-
-        let account = AccountData { free: U256::MAX };
-        Account::<MockRuntime>::insert(&asset_id, &bob, &account);
-
-        assert_noop!(
-            Asset::transfer(
-                Origin::signed(alice.clone()),
-                asset_id,
-                bob.clone(),
-                1.into()
-            ),
-            Error::<MockRuntime>::FreeTransferOverflow,
-        );
-    });
-}
+// TODO Trello 91 - Enable commented out tests
+
+// use crate::mock::{new_tester, AccountId, Asset, MockRuntime, Origin, System, TestEvent};
+//
+// use frame_support::storage::StorageDoubleMap;
+// use frame_support::{assert_noop, assert_ok};
+// use sp_core::H160;
+// use sp_keyring::AccountKeyring as Keyring;
+//
+// use crate::{Account, TotalIssuance};
+//
+// use super::*;
+//
+// fn last_event() -> TestEvent {
+//     System::events().pop().expect("Event expected").event
+// }
+//
+// fn set_balance<T>(asset_id: &H160, account_id: &AccountId, amount: T)
+// where
+//     T: Into<U256> + Copy,
+// {
+//     let free = amount.into();
+//     let account = AccountData { free };
+//     Account::<MockRuntime>::insert(&asset_id, &account_id, &account);
+//     TotalIssuance::insert(&asset_id, free);
+// }
+//
+// #[test]
```

Comment:
these tests of bridged asset are no longer relevant since token functionality like mint, burn, and transfer is no longer implemented in bridged asset, it is no longer responsible for that functionality. Thus, these tests are to be removed.